net-nntp/slrn/slrn-1.0.3.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_P="${P}a"
```

Comment:
This `a` is only present in the tarball name. Once extracted, the directory is called `slrn-1.0.3`, without `a` and all communications are made without the trailing `a`. I don't know why there is this `a` in the tarball name...

Debian came back from using that `a` as a version identifier: https://sources.debian.net/src/slrn/1.0.3%2Bdfsg-1/debian/changelog/